src-built-in/components/advancedAppLauncher/src/components/FoldersList.jsx

```diff
@@ -177,8 +178,14 @@ export default class FoldersList extends React.Component {
 				isNameError: true
 			});
 		}
+
+		// Check if the submission is the same text as the old name.
+		// If false, renaming will be skipped
+		let nameChanged = true;
+		if (oldName.trim() === newName.trim()) nameChanged = false;
+
 		// Names must be unique, folders cant share same names
-		if (folders[newName]) {
+		if (folders[newName] && nameChanged) {
```

Comment:
Is this check even needed now? The only way to have folders with numbers automatically added will be clicking new folder now.